tests/fixtures/textarea-form-field.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Form with ChoiceFormField</title>
```

Comment:
Title does not describe them properly

Reply:
You are right. I need to create a new PR because this one is closed, am I right?

Reply:
I'll take care of this

Reply:
Fixed by 7a041cb.

Reply:
but this PR is *breaking* these types (as they were all considered textual before)